### PR TITLE
YM-191 | Update HDS

### PR DESCRIPTION
**Pull request by tuovinensanttu**
Includes tickets:
- YM - 197
- YM - 187

- Changed all HDS variables.
- Replaced Select component with same one that is used in profile and jässäri-admin.
- Created LinkButton that takes styles from HDS.
- Various UI fixes.

### CODE REVIEW

File: src/common/layout/PageLayout.module.css [hunk 1]
```diff
@@ -19,16 +13,24 @@ header {
...
+        background-size: 720px auto;
+    }
 }
 
-.youthBackground {
-    background-color: #ffc61e;
+
+@media (min-width: 800px) {
+    .hostingBox {
+        margin: 20px auto 0;
```
Comment (Jokinen) [inline src/common/layout/PageLayout.module.css:33]: No HDS token here?

File: src/pages/membership/components/membershipDetails/MembershipDetails.module.css [hunk 2]
```diff
@@ -39,12 +39,22 @@
...
-  margin-top: 40px;
-  padding: 16px 50px;
+.button {
   margin-right: 20px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.membershipDetails h1 {
+  font-size: var(--fontsize-h-2);
```
Comment (badrange) [inline src/pages/membership/components/membershipDetails/MembershipDetails.module.css:49]: This is interesting! Is this a symptom of a problem with hds' typography?

File: src/pages/membership/components/membershipInformation/MembershipInformation.tsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ import {
 import styles from './MembershipInformation.module.css';
 import getFullName from '../../helpers/getFullName';
 import convertDateToLocale from '../../helpers/convertDateToLocale';
+import './../../../../../node_modules/hds-core/lib/components/button/button.css';
```
Comment (badrange) [inline src/pages/membership/components/membershipInformation/MembershipInformation.tsx:19]: Is there a specific reason for having a relative import?

File: src/pages/membership/components/youthProfileForm/YouthProfileForm.module.css [hunk 4]
```diff
@@ -1,10 +1,6 @@
-.formTitleText {
-    text-align: center;
-}
-@media (min-width: 768px) {
-    .formTitleText {
-        text-align: left;
-    }
+.formTitleText h1 {
+    font-size: var(--fontsize-h-2);
```
Comment (badrange) [inline src/pages/membership/components/youthProfileForm/YouthProfileForm.module.css:2]: Again: Is the HDS h1 way too big or what is going on.. 🤔

File: src/common/linkButton/LinkButton.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+
+import '../../../node_modules/hds-core/lib/components/button/button.css';
```
Comment (badrange) [inline src/common/linkButton/LinkButton.tsx:5]: Why relative import instead of absolute?

File: package.json [hunk 6]
```diff
@@ -30,8 +30,8 @@
     "formik": "^2.0.3",
     "graphql": "^14.5.8",
     "graphql.macro": "^1.4.2",
-    "hds-core": "^0.4.1",
-    "hds-react": "^0.5.4",
+    "hds-core": "^0.6.0",
```
Comment (badrange) [inline package.json:33]: I think 0.6.2 is out already :-)

**Pull request merged**